src/main/scala/io/gatling/sbt/settings/gatling/EnterpriseSettings.scala

```diff
@@ -265,13 +265,14 @@ object EnterpriseSettings {
 
   private def interactiveCreateOrStartEnterpriseSimulation(config: Configuration) = Def.task {
     val logger = streams.value.log
+    val optionalTeamId = configOptionalString(config / enterpriseTeamId).value.map(UUID.fromString)
+    val optionalSimulationClass = configOptionalString(config / enterpriseSimulationClass).value
+    val classNames: Seq[String] = (config / definedTests).value.map(_.name)
+    val file = buildEnterprisePackage(config).value
```

Comment:
(only comment here, but it's a general comment)

Does that mean we should define dynTask?

Reply:
We do statically know which tasks we depend on, so I don't see a reason to use a dynTask?